### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by trianglesphere**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,24 +1,24 @@
 * @protolambda @tynes @ajsutton @sebastianst @mslipper
 
-batcher.md @sebastianst @trianglesphere
+batcher.md @sebastianst @protolambda
```
Comment (sebastianst) [inline .github/CODEOWNERS]: can do @axelKingsley instead

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 # overview.md
 predeploys.md @tynes
 proposals.md @ajsutton @clabby @tynes @maurelian
-rollup-node-p2p.md @sebastianst @trianglesphere
-rollup-node.md @sebastianst @trianglesphere
+rollup-node-p2p.md @sebastianst @protolambda
+rollup-node.md @sebastianst @protolambda
 safe-liveness-checking.md @maurelian @mds1
-span-batches.md @sebastianst @trianglesphere
+span-batches.md @sebastianst @protolambda
```
Comment (sebastianst) [inline .github/CODEOWNERS]: can do @axelKingsley instead

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,24 +1,24 @@
 * @protolambda @tynes @ajsutton @sebastianst @mslipper
 
-batcher.md @sebastianst @trianglesphere
+batcher.md @sebastianst @protolambda
 bridges.md @tynes @maurelian
 deposits.md @protolambda @tynes
-derivation.md @protolambda @trianglesphere
-exec-engine.md @sebastianst @trianglesphere
+derivation.md @protolambda @sebastianst
+exec-engine.md @sebastianst @protolambda
```
Comment (sebastianst) [inline .github/CODEOWNERS]: maybe @ajsutton and/or @axelKingsley for additional coverage?

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 guaranteed-gas-market.md @tynes @maurelian
 # introduction.md
 messengers.md @tynes @maurelian
 # overview.md
 predeploys.md @tynes
 proposals.md @ajsutton @clabby @tynes @maurelian
-rollup-node-p2p.md @sebastianst @trianglesphere
-rollup-node.md @sebastianst @trianglesphere
+rollup-node-p2p.md @sebastianst @protolambda
+rollup-node.md @sebastianst @protolambda
```
Comment (sebastianst) [inline .github/CODEOWNERS]: we can also add @axelKingsley here

File: .github/CODEOWNERS (hunk 1 above)
Comment (trianglesphere) [inline .github/CODEOWNERS]:
```suggestion
batcher.md @axelKingsley @protolambda
```

File: .github/CODEOWNERS (hunk 3 above)
Comment (trianglesphere) [inline .github/CODEOWNERS]:
```suggestion
derivation.md @protolambda @sebastianst @ajsutton 
exec-engine.md @protolambda @sebastianst @ajsutton
```

File: .github/CODEOWNERS (hunk 4 above)
Comment (trianglesphere) [inline .github/CODEOWNERS]:
```suggestion
rollup-node.md @sebastianst @protolambda @axelKingsley 
```

File: .github/CODEOWNERS (hunk 2 above)
Comment (trianglesphere) [inline .github/CODEOWNERS]:
```suggestion
span-batches.md @sebastianst @protolambda @axelKingsley 
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,24 +1,24 @@
 * @protolambda @tynes @ajsutton @sebastianst @mslipper
 
-batcher.md @sebastianst @trianglesphere
+batcher.md @axelKingsley @protolambda
```
Comment (trianglesphere) [inline .github/CODEOWNERS]:
```suggestion
batcher.md @sebastianst @axelKingsley
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
 # overview.md
 predeploys.md @tynes
 proposals.md @ajsutton @clabby @tynes @maurelian
-rollup-node-p2p.md @sebastianst @trianglesphere
-rollup-node.md @sebastianst @trianglesphere
+rollup-node-p2p.md @sebastianst @protolambda
+rollup-node.md @sebastianst @protolambda @axelKingsley 
 safe-liveness-checking.md @maurelian @mds1
-span-batches.md @sebastianst @trianglesphere
+span-batches.md @sebastianst @protolambda @axelKingsley
```
Comment (sebastianst) [inline .github/CODEOWNERS]:
```suggestion
span-batches.md @sebastianst @axelKingsley 
```

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -1,24 +1,24 @@
...
 # glossary.md
 guaranteed-gas-market.md @tynes @maurelian
 # introduction.md
 messengers.md @tynes @maurelian
 # overview.md
 predeploys.md @tynes
 proposals.md @ajsutton @clabby @tynes @maurelian
-rollup-node-p2p.md @sebastianst @trianglesphere
-rollup-node.md @sebastianst @trianglesphere
+rollup-node-p2p.md @sebastianst @protolambda
```
Comment (sebastianst) [inline .github/CODEOWNERS]:
```suggestion
rollup-node-p2p.md @sebastianst @protolambda @axelKingsley
```

File: .github/CODEOWNERS [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
+rollup-node.md @sebastianst @protolambda @axelKingsley 
 safe-liveness-checking.md @maurelian @mds1
-span-batches.md @sebastianst @trianglesphere
+span-batches.md @sebastianst @protolambda @axelKingsley 
 superchain-configuration.md @protolambda @tynes
 superchain-upgrades.md @protolambda @tynes
-system_config.md @sebastianst @trianglesphere
+system_config.md @sebastianst @protolambda
```
Comment (sebastianst) [inline .github/CODEOWNERS]:
```suggestion
system_config.md @sebastianst @protolambda @tynes
```

**Review by sebastianst — APPROVED** [review]

**Pull request merged**